doc/man3/EVP_bf.pod

```diff
@@ -0,0 +1,61 @@
+=pod
+
+=head1 NAME
+
+EVP_bf_cbc,
+EVP_bf_cfb,
+EVP_bf_ecb,
+EVP_bf_ofb
+- EVP Blowfish cipher
+
+=head1 SYNOPSIS
+
+=for comment generic
+
+ #include <openssl/evp.h>
+
+ const EVP_CIPHER *EVP_ciphername(void)
```

Comment:
Since there is only 4, I'd just list them. Same comment applies for all the other pages where there are 4 or less of these.

Reply:
I could go for 10 ;-)

Reply:
10 works too!

Reply:
I've used the less-than-10 rule to elaborate out the functions names 😉